docs/examples/resnet50_trt/README.md

```diff
@@ -0,0 +1,255 @@
+# Triton Inference Server with DALI backend
+
+This repo is an example of [DALI (Data Loading Library) backend](https://github.com/NVIDIA/DALI) for image classification on [Triton Inference server](https://github.com/triton-inference-server/server).
+
+ResNet50 model optimized by [TensorRT](https://developer.nvidia.com/tensorrt) is used for image classification. 
+
+
+#### Dependencies
+
+* Export ONNX and build TensorRT
+  * `nvcr.io/nvidia/pytorch:20.12-py3`
+* Triton Inference Server for DALI backend
+  * `nvcr.io/nvidia/tritonserver:20.12-py3`
+  * `https://github.com/triton-inference-server/dali_backend`
+
+* Client
+  * `nvcr.io/nvidia/tritonserver:20.10-py3-client`
+
+## Setting up the ONNX-TensorRT DALI
+
+The quickest way to set up this example is to run the setup script:
+
+```
+cd dali_backend/docs/examples/resnet50_trt
+sh setup_resnet50_trt_example.sh
+```
+
+Below we add step-by-step guide, how to set up and run this example.
+If you used the `setup_resnet50_trt_example.sh` script, please skip to
+[Run Triton Inference Server](#run-triton-inference-server) section.
+
+### Step-by-step guide
+
+Prepare directory structure for the model repository.
+
+```
+mkdir -p model_repository/dali/1
+mkdir -p model_repository/ensemble_dali_resnet50/1
+mkdir -p model_repository/resnet50_trt/1
+```
+
+####  Build TensorRT via ONNX
+
+##### 1.  Converting PyTorch Model to ONNX-model 
+
+Run `onnx_exporter.py` for conversion using PyTorch model to ONNX model. In this case, ResNet50 model is converted to ONNX format. `width` and `height` dims are fixed at 224 but dynamic axes arguments for dynamic batch is used. 
+
+```
+docker run -it --gpus=all -v $(pwd):/workspace nvcr.io/nvidia/pytorch:20.12-py3 bash
+python onnx_exporter.py --save model.onnx
+```
+
+##### 2. Building ONNX-model to TensorRT engine
+
+Set the arguments for enabling fp16 precision `--fp16` and for dynamic shapes using a profile `--minShapes`, `--optShapes`, and `maxShapes` with `--explicitBatch`
+
+```python
+trtexec --onnx=model.onnx --saveEngine=./model_repository/resnet50_trt/1/model.plan --explicitBatch --minShapes=input:1x3x224x224 --optShapes=input:1x3x224x224 --maxShapes=input:256x3x224x224 --fp16
+```
+
+##### 3. Serialize DALI pipeline 
+
+Run `serialize_dali_pipeline.py` for generating  DALI pipeline. 
+
+```
+python serialize_dali_pipeline.py --save ./model_repository/dali/1/model.dali
+```
+
+If had a dedicated hardware decoder, the hardware is available. Set the preprocessing pipeline  `resize` and `normalize` using `dali.pipeline.Pipeline` and serialize pipeline using `Pipeline.serialize`
```

Comment:
?

Reply:
I decided to get rid of the snippet, that shows DALI pipeline. This code is shown in repo and easily found. I'll add short info to this paragraph